srcpkgs/wlroots/template

```diff
@@ -1,8 +1,7 @@
 # Template file for 'wlroots'
 pkgname=wlroots
-reverts="0.6.0_1"
-version=0.5.0
-revision=2
+version=0.6.0
+revision=1
```

Comment:
We already had `wlroots-0.6.0_1` by accident. You have to set revision to 2 to avoid confusion.

Reply:
Done